src/filters/string.ts

```diff
@@ -4,6 +4,8 @@
  * * prefer stringify() to String() since `undefined`, `null` should eval ''
  */
 import { assert, escapeRegExp, stringify } from '../util'
+import { marked } from 'marked'
+import { sanitize } from 'isomorphic-dompurify'
```

Comment:
currently liquidjs has only one run time dependency `commander` and that is not included in dist files. Let's avoid introducing dependencies.

Markdown functionality is better to be maintained in another repo, maybe liquidjs-plugin-markdown, or liquidjs-plugin-jekyll.